lms/djangoapps/support/views/course_entitlements.py

```diff
@@ -6,20 +6,43 @@
 from django.db.models import Q
 from django.http import HttpResponseBadRequest
 from django.utils.decorators import method_decorator
+from django.views.generic import View
 from edx_rest_framework_extensions.authentication import JwtAuthentication
 from rest_framework import permissions, status, viewsets
 from rest_framework.response import Response
 
+from edxmako.shortcuts import render_to_response
 from entitlements.api.v1.permissions import IsAdminOrAuthenticatedReadOnly
 from entitlements.api.v1.serializers import SupportCourseEntitlementSerializer
 from entitlements.models import CourseEntitlement, CourseEntitlementSupportDetail
+from lms.djangoapps.commerce.utils import EcommerceService
 from lms.djangoapps.support.decorators import require_support_permission
 from openedx.core.djangoapps.cors_csrf.authentication import SessionAuthenticationCrossDomainCsrf
 
 REQUIRED_CREATION_FIELDS = ['course_uuid', 'reason', 'mode']
 
 
-class EntitlementSupportView(viewsets.ModelViewSet):
+class EntitlementSupportView(View):
+    """
+    View for viewing and changing learner enrollments, used by the
+    support team.
+    """
+    @method_decorator(require_support_permission)
+    def get(self, request):
+        """Render the enrollment support tool view."""
+        support_actions = CourseEntitlementSupportDetail.get_support_actions_list()
```

Comment:
Nit: Purely a personal style preference - you could consider assigning these variable references to their relevant `context` keys directly instead of assigning them to a variable, then assigning that variable to the key.

Doesn't really matter, and you might run into `pylint` errors around line-length ( 😞), but basically I hate variables so...

Reply:
Yea.... its a combo of line length and I like keeping the context object instantiation pretty clean (i.e. minimal function calls in it).